src/Mod/Sketcher/Gui/SketcherSettingsGrid.ui

```diff
@@ -0,0 +1,461 @@
+           </property>
+           <property name="minimum">
+            <number>1</number>
+           </property>
+           <property name="maximum">
+            <number>99</number>
+           </property>
+           <property name="value">
+            <number>2</number>
+           </property>
+           <property name="prefEntry" stdset="0">
+            <cstring>GridDivLineWidth</cstring>
+           </property>
+           <property name="prefPath" stdset="0">
+            <cstring>Mod/Sketcher/General</cstring>
+           </property>
+          </widget>
+         </item>
+         <item row="3" column="0">
+          <widget class="QLabel" name="label_gridDivLineColor">
+           <property name="text">
+            <string>Line color</string>
+           </property>
+           <property name="buddy">
+            <cstring>gridDivLineColor</cstring>
+           </property>
+          </widget>
+         </item>
+         <item row="3" column="1">
+          <widget class="Gui::PrefColorButton" name="gridDivLineColor">
+           <property name="sizePolicy">
+            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
+             <horstretch>0</horstretch>
+             <verstretch>0</verstretch>
+            </sizepolicy>
+           </property>
+           <property name="text">
+            <string/>
+           </property>
+           <property name="color">
+            <color>
+             <red>178</red>
+             <green>178</green>
+             <blue>178</blue>
+            </color>
+           </property>
+           <property name="prefEntry" stdset="0">
+            <cstring>GridDivLineColor</cstring>
+           </property>
+           <property name="prefPath" stdset="0">
+            <cstring>Mod/Sketcher/General</cstring>
+           </property>
+          </widget>
+         </item>
+ 	    </layout>
+       </widget>
+      </item>
+	 </layout>
+    </widget>
+   </item>
+   <item>
+    <spacer name="verticalSpacer">
+     <property name="orientation">
+      <enum>Qt::Vertical</enum>
+     </property>
+     <property name="sizeHint" stdset="0">
+      <size>
+       <width>20</width>
+       <height>40</height>
+      </size>
+     </property>
+    </spacer>
+   </item>
+  </layout>
+ </widget>
+ <customwidgets>
+  <customwidget>
+   <class>Gui::PrefSpinBox</class>
+   <extends>QSpinBox</extends>
+   <header>Gui/PrefWidgets.h</header>
+  </customwidget>
+  <customwidget>
+   <class>Gui::PrefCheckBox</class>
+   <extends>QCheckBox</extends>
+   <header>Gui/PrefWidgets.h</header>
+  </customwidget>
+  <customwidget>
+   <class>Gui::PrefQuantitySpinBox</class>
+   <extends>Gui::QuantitySpinBox</extends>
+   <header>Gui/PrefWidgets.h</header>
+  </customwidget>
+  <customwidget>
+   <class>Gui::PrefColorButton</class>
+   <extends>Gui::ColorButton</extends>
+   <header>Gui/PrefWidgets.h</header>
+  </customwidget>
+ </customwidgets>
```

Comment:
I think there are some missings here as idea is to go down to Qt native widgets
```
  <customwidget>
   <class>Gui::QuantitySpinBox</class>
   <extends>QAbstractSpinBox</extends>
   <header>Gui/QuantitySpinBox.h</header>
  </customwidget>
```
and
```
  <customwidget>
   <class>Gui::ColorButton</class>
   <extends>QPushButton</extends>
   <header>Gui/Widgets.h</header>
  </customwidget>
```

Reply:
Did you meant PrefQuantitySpinBox ? Or as you wrote QuantitySpinBox ?

Reply:
As I wrote. :)